admin_guide/manage_users.adoc

```diff
@@ -9,84 +9,77 @@
 
 toc::[]
 
+[[managing-users-overview]]
+== Overview
+
+This topic describes the management of user accounts, including how new user
+accounts are created in {product-title} and how they can be deleted.
+
+[[managing-users-adding-users]]
 == Adding Users
 
-Once new users log in to OpenShift, an account is created for that user and the
-cluster administrator can
+After new users log in to {product-title}, an account is created for that user
+and the cluster administrator can
```

Comment:
Maybe link to the config_auth topic: https://docs.openshift.com/enterprise/latest/install_config/configuring_authentication.html#identity-providers

Something like, "After new users log in to OpenShift, an account is created per the identity provider configured on the master."